client/resources/original_messages.json

```diff
@@ -644,20 +644,20 @@
     "description": "Affirmative answer to a form question.",
     "message": "Yes"
   },
-  "theme_dark": {
-    "description": "The text for a main menu item that takes the user to a page to set the application theme to dark mode.",
+  "appearance_dark": {
```

Comment:
Please make sure that any feature that depends on translations don't block releases, as we plan to do weekly builds for QA. They need to be gated on the existence of the translations.
/cc @jyyi1